compiler/rustc_middle/src/ty/sty.rs

```diff
@@ -2535,6 +2545,8 @@ impl<'tcx> Ty<'tcx> {
                 field_tys.len() <= 3 && field_tys.iter().all(Self::is_trivially_pure_clone_copy)
             }
 
+            ty::Pat(ty, _) => ty.is_trivially_pure_clone_copy(),
```

Comment:
This disagrees with the solver, since I can't seem to find builtin impls (only looked in the new solver).

Reply:
Nvm, only new solver I think. Should be fixed.